app/services/model_wizard.rb

```diff
@@ -0,0 +1,52 @@
+class ModelWizard
+  attr_reader :object
+
+  def initialize(object, session, params=nil, object_params=nil)
+    @object = object
```

Comment:
Can we call `@object` something else? It gets a bit confusing as well since `@object` starts out as the step name and then becomes an object